badparking/core/models.py

```diff
@@ -11,3 +13,23 @@ def __unicode__(self):
     class Meta:
         verbose_name = "Тип порушення"
         verbose_name_plural = "Типи порушень"
+
+
+class Claim(models.Model):
+    license_plates = models.CharField("Номери автомобілів", max_length=50, blank=True)
+    longitude = models.CharField("Довгота", max_length=50, blank=True)
+    latitude = models.CharField("Широта", max_length=50, blank=True)
+    city = models.CharField("Місто", max_length=30, blank=True)
+    address = models.CharField("Адреса", max_length=150, blank=True)
+    created_at = models.DateTimeField(editable=False, auto_now_add=True)
+
+    user = models.ForeignKey(User, verbose_name="Користувач")
```

Comment:
We'll be using a custom User model. Also it's a generally better practice to reference `settings.AUTH_USER_MODEL` instead of a concrete model in foreign keys. Looking at migrations Django did the right thing there but there still might be runtime issues.